services/tables/src/main/java/com/linkedin/openhouse/tables/config/MainApplicationConfig.java

```diff
@@ -41,6 +48,10 @@ public class MainApplicationConfig extends BaseApplicationConfig {
   private static final Pattern VERSION_PART_PATTERN = Pattern.compile("v[0-9]+");
   private static final int IN_MEMORY_BUFFER_SIZE = 10 * 1000 * 1024;
 
+  public static final String HEADER_X_CLIENT_NAME = "X-Client-Name";
+  public static final String METRIC_KEY_CLIENT_NAME = "client_name";
+  public static final List<String> ALLOWED_CLIENT_NAMES = Arrays.asList("trino", "spark");
```

Comment:
what about catalog in a normal JVM, like workflow-scheduler's container? 
we've been shipping catalog in these environments and we can preset these values as part of shipped lib.